api/app.js

```diff
@@ -1,21 +1,35 @@
 const createError = require('http-errors')
+const mongoose = require('mongoose');
 const express = require('express')
 const path = require('path')
+const app = express()
```

Comment:
Not sure what the general convention on this is, but I think I'd prefer this variable declaration to be after all the other imports.

Reply:
fixed